### ISSUE TITLE
A11y_Code.Visualstudio.Com(CELA-P1)_Updates-November 2018_Keyboard:# Symbol is not accessible using keyboard to the user,When we mouse hover,# symbol is visible and accessible using Mouse.

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-docs#2654]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
**Environment Details:**
https://code.visualstudio.com/updates/v1_30

Note:
This issue is present in the below pages in the application.
code.visualstudio.com/docs/supporting/faq
code.visualstudio.com/docs/setup/linux
code.visualstudio.com/docs/getstarted/settings
code.visualstudio.com/docs/python/python-tutorial
code.visualstudio.com/updates/v1_30
code.visualstudio.com/docs/editor/whyvscode
code.visualstudio.com/docs/getstarted/tips-and-tricks
code.visualstudio.com/updates/v1_29
code.visualstudio.com/docs/editor/integrated-terminal
code.visualstudio.com/docs/editor/debugging
code.visualstudio.com/docs/editor/versioncontrol
​
**Version:​**
Microsoft Edge 44.18362.1.0
Version 74.0.3729.131 (Official Build) (64-bit)
keyboard
​
**Repro Steps:​**
1)Hit the URL - code.visualstudio.com/updates/v1_30
2)Tab till Workbench, Multiline Search Input, Custom title and menu bar on Linux by default
3)Check whether # symbol is accessible using keyboard or not as it is accessible using mouse.

**Actual Result:​**
The # symbol is not accessible using keyboard to the user, When we mouse hover,# symbol is visible and accessible to the user using mouse.
​
**Expected Result:​**
All the actionable buttons and interactable controls which are accessible using mouse should be accessible using keyboard also.​
​
**User Impact:​**
Visually impaired, Low vision and People with hand tremors will have difficulty if the button or controls are not accessible using keyboard​.

​​MAS Reference:​
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Comment by Ashwinipola26** [comment]
[bug_2654_# Symbol is not accessible using keyboard to the user -10 MB.pptx](https://github.com/microsoft/vscode-docs/files/3161990/bug_2654_.Symbol.is.not.accessible.using.keyboard.to.the.user.-10.MB.pptx)

**Comment by Ashwinipola26** [comment]
**Environment Details:**
code.visualstudio.com/updates/v1_30
​
**Version:​**
Microsoft Edge 44.18362.1.0
Version 74.0.3729.131 (Official Build) (64-bit)
​
**Repro Steps:​**
1)Hit the URL - code.visualstudio.com/updates/v1_30
2)Tab till Workbench, Multiline Search Input, Custom title and menu bar on Linux by default
3)Check whether # symbol is accessible using keyboard or not as it is accessible using mouse.
Actual Result:​
Symbol is not accessible using keyboard to the user, When we mouse hover,# symbol is visible and accessible to the user using mouse.
​
**Expected Result:​**
All the actionable buttons and interactable controls which are accessible using mouse should be accessible using keyboard also.​
​
**User Impact:​**
Visually impaired, Low vision and People with hand tremors will have difficulty if the button or controls are not accessible using keyboard​.
[bug_2654_# Symbol is not accessible using keyboard to the user -10 MB.pptx](https://github.com/Microsoft/vscode-docs/files/3165084/bug_2654_.Symbol.is.not.accessible.using.keyboard.to.the.user.-10.MB.pptx)

​​**MAS Reference:​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Comment by Ashwinipola26** [comment]
[bug_2654_# Symbol is not accessible using keyboard to the user -10 MB.pptx](https://github.com/microsoft/vscode-docs/files/3165246/bug_2654_.Symbol.is.not.accessible.using.keyboard.to.the.user.-10.MB.pptx)

**Comment by Ashwinipola26** [comment]
[bug_2654_# symbol is not accessible using keyboard_01.pptx](https://github.com/microsoft/vscode-docs/files/3172063/bug_2654_.symbol.is.not.accessible.using.keyboard_01.pptx)

**Comment by Ashwinipola26** [comment]
[bug_2654_# symbol is not accessible using keyboard_03.pptx](https://github.com/microsoft/vscode-docs/files/3172065/bug_2654_.symbol.is.not.accessible.using.keyboard_03.pptx)

**Comment by Ashwinipola26** [comment]
[bug_2654_# symbol is not accessible using keyboard_04.pptx](https://github.com/microsoft/vscode-docs/files/3172068/bug_2654_.symbol.is.not.accessible.using.keyboard_04.pptx)

**Comment by auchenberg** [comment]
My interoperation: The headers with the hover over hash link should have a tab index, so if you tab through focusable elements it should stop at each of the headers, like if you hovered over the element had saw the hashlink.

**Comment by gregvanl** [comment]
This was added as an optional way to copy the h2 and h3 links for bookmarking. We don't want to add another tab stop for the link as it will make header navigation awkward. Keyboard users can get the same h2, h3 link URLs from the browser URL so there is no missing functionality.

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
